.github/workflows/ci-check.yml

```
@@ -9,8 +9,13 @@ on:
       - 'renovate/**'
   pull_request:
     paths:
+      # only runs this CI when changes are in 'docs/**' or 'examples/**'
+      # the negate(!) is necessary, otherwise the CI runs when the changes are and are not in 'docs/**' or 'examples/**'
+      # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths
       - 'docs/**'
+      - '!docs/**'
       - 'examples/**'
+      - '!examples/**'
```

```suggestion
      - 'docs/**'
      - 'examples/**'
      - '!framer/**'
      - '!modules/**'
      - '!netlify/**'
      - '!packages/**'
      - '!scripts/**'
      - '!test/**'
```
@michaldudak I could not find a better option for built-in github actions path filtering 🥲.

---

Have you verified it on a separate repo?

---

https://github.com/siriwatknp/poc-github-actions/pull/13 it does not work 🥲

Now, it's clear to me https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths.
```
This example runs anytime the push event includes a file in the sub-project directory or its subdirectories, unless the file is in the sub-project/docs directory. For example, a push that changed sub-project/index.js or sub-project/src/index.js will trigger a workflow run, but a push changing **only** sub-project/docs/readme.md will not.
```